model_compression_toolkit/gptq/keras/gptq_training.py

```diff
@@ -169,16 +162,17 @@ def get_activation_quantizer_holder(self, n: common.BaseNode) -> Union[None, Cal
         Returns:
             A ActivationQuantizationHolder layer for the node activation quantization.
         """
-        if self._is_gptq_applicable(n): # If it was wrapped before - add holder instead
-            _, activation_quantizers = quantization_builder(n, self.gptq_config)
-            # Holder by defenition uses a single quantizer for the activation quantization
-            # thus we make sure this is the only possible case (unless it's a node we no activation
-            # quantization, which in this case has an empty list).
-            if len(activation_quantizers) > 0:
-                assert len(activation_quantizers) == 1, f'ActivationQuantizationHolder supports a ' \
-                                                        f'single quantizer but {len(activation_quantizers)} quantizers were found for node {n}'
-                return ActivationQuantizationHolder(activation_quantizers[0])
-        return None
+        _, activation_quantizers = quantization_builder(n, self.gptq_config)
+        # Holder by definition uses a single quantizer for the activation quantization
+        # thus we make sure this is the only possible case (unless it's a node we no activation
+        # quantization, which in this case has an empty list).
+        if len(activation_quantizers) > 0:
+            assert len(activation_quantizers) == 1, f'ActivationQuantizationHolder supports a ' \
+                                                    f'single quantizer but {len(activation_quantizers)} quantizers were found for node {n}'
+            return ActivationQuantizationHolder(activation_quantizers[0])
+
+        # Return identity function if no quantization is needed
+        return lambda x: x
```

Comment:
Let's go over this one offline again, I think it's a bad idea to have such a hidden "solution" in our code, to a problem that was created by our design.